manifests/z/ZeroTier/ZeroTierOne/1.8.2/ZeroTier.ZeroTierOne.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ZeroTier.ZeroTierOne
+PackageVersion: 1.8.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /quiet
+UpgradeBehavior: install
+ProductCode: '{2B153F53-2BD3-42C7-9972-D57F94E3CB8D}'
```

Comment:
I feel like this should be removed from the spec in a breaking version 2.0. The likelihood of n>1 installer entries having the same product code is so low that putting ProductCode outside the installer entry is just there to trip up people not using `wingetcreate` or `YamlCreate.ps1`.

Reply:
@Trenly 
Maybe we could put MSI ProductCode's inside the Installer since they change version from version?
Does Burn also change ProductCode for each version?

Imo having the ProductCode outside of the Installer is a lot better for Static ProductCodes, but dynamic ones i can understand being under the installer

Reply:
Some of us wrote automation that wasn't aware that this could even happen ;)

Reply:
Static ProductCodes are often used for troubled .exe’s and i’m even seeing it correctly used by some rookies (when created in YamlCreate 🤘), which reminds me that the ProductCode missmatch is a RARE issue mostly caused by wingetbot, first time submition or hardheaded users…

Also we’re just following the JSON schema 🤷‍♂️ (Which is at the top of every manifest) Not the REST schema that you like to reference, but i guess we can blame not syncronized ducumentation for that.

Luckily any issues on our end can quickly get patched.